examples/arm/ethos-u-setup/core_platform/patches/0008-Use-new-LIB_ET_OP_REGISTRATION-lib-path.patch

```diff
@@ -0,0 +1,24 @@
+From 64cda3dc786f2436177de7ee8805526e216afa74 Mon Sep 17 00:00:00 2001
+From: Hansong Zhang <hsz@meta.com>
+Date: Wed, 1 Nov 2023 22:03:53 -0700
+Subject: [PATCH] Use new LIB_ET_OP_REGISTRATION lib path
+
+---
+ applications/executorch_tests/CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/applications/executorch_tests/CMakeLists.txt b/applications/executorch_tests/CMakeLists.txt
+index 5f70e8e..a07f22f 100644
+--- a/applications/executorch_tests/CMakeLists.txt
++++ b/applications/executorch_tests/CMakeLists.txt
+@@ -43,7 +43,7 @@ message("**********************")
+
+ set(LIB_ET_RUNTIME "${ET_BUILD_DIR_PATH}/libexecutorch.a")
+ set(LIB_ET_ETHOS "${ET_BUILD_DIR_PATH}/backends/arm/libexecutorch_delegate_ethos_u.a")
+-set(LIB_ET_OP_REGISTRATION "${ET_BUILD_DIR_PATH}/kernels/portable/libportable_ops_lib.a")
++set(LIB_ET_OP_REGISTRATION "${ET_BUILD_DIR_PATH}/examples/arm/libarm_ops_lib.a")
```

Comment:
not changing this?

Reply:
Let's do it later. I have trouble with docker pulling this change